lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -80,7 +80,7 @@ def init_from_structure(self, student, course_structure, scores_client, submissi
                 student, descendant_key, course_structure, scores_client, submissions_scores, persisted_values={},
             )
         self.all_total, self.graded_total = graders.aggregate_scores(self.scores, self.display_name, self.location)
-        self._log_event(log.warning, u"init_from_structure", student)
+        self._log_event(log.info, u"init_from_structure", student)
```

Comment:
In order to reduce the amount of noise in splunk logs, I dropped this, `init_from_model`, and `SGF.create` down to info level, as they seem to be the types of events that will pile up very quickly.

<img width="1432" alt="screen shot 2016-09-20 at 3 57 33 pm" src="https://cloud.githubusercontent.com/assets/7373924/18686757/149e559a-7f4b-11e6-821e-24b435f0c205.png">